proxies/live/apiproxy/policies/ServiceCallout.ValidateAccessToken.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ServiceCallout async="false" continueOnError="true" enabled="true" name="ServiceCallout.ValidateAccessToken">
+    <DisplayName>ServiceCallout.ValidateAccessToken</DisplayName>
+    <Properties/>
+    <Request clearPayload="true" variable="tokenValidationRequest">
+        <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
+        <Set>
+            <Headers>
+                <Header name="GPC-Authorization">{request.header.GPC-Authorization}</Header>
+            </Headers>
+            <Verb>POST</Verb>
+        </Set>
+    </Request>
+    <Response>tokenValidationResponse</Response>
+    <HTTPTargetConnection>
+        <URL>https://{endpoint}/validate</URL>
```

Comment:
Please rename this var. `endpoint` seems like you're changing the proxy endpoint but it's just the token validation.

Reply:
Also the endpoint of the validate is not necessarly the same as proxy endpoint. In case of aws dev it is but in prod, provider has a different url that the auth server. So it's better to treat these urls in separate variables